JASP-Desktop/html/js/utils.js

```diff
@@ -374,22 +352,8 @@ function formatColumn(column, type, format, alignNumbers, combine, modelFootnote
 			else {
 				var strContent;
 
-				if (content < 1/(Math.pow(10,dp))) {
-					let _sign = (html) ? "&minus;" : "-";
-					var exponentiated = content.toExponential(dp).replace(/-/g, _sign)
-
-					var split = exponentiated.split("e")
-					var mantissa = split[0]
-					var exponent = split[1]
-					var exponentSign = exponent.substr(0, 1)
-					var exponentNum = exponent.substr(1)
-
-					if (html) {
-						strContent = mantissa + "e&thinsp;" + exponentSign + exponentNum;
-					} else {
-						strContent = mantissa + "e" + exponentSign + exponentNum;
-					}
-
+				if (content != 0 && Math.abs(content) < 1/(Math.pow(10,dp))) {
```

Comment:
This creates the edge case that `-0.001` will be displayed as `-1.000e-3`. But whatever I don't think this is a big deal